src/routes/paths.ts

```diff
@@ -67,6 +67,7 @@ export const paths = {
   organization: (orgId: string) => `/organizations/${orgId}`,
   organizationSettings: (orgId: string) => `/organizations/${orgId}/settings`,
   vaults: '/vaults',
+  vaultsForOrg: (orgId: string) => `/vaults?org=${orgId}`,
```

Comment:
I'd prefer a more sturcutred URL schema for orgs like:

instead of:
http://localhost:3000/vaults?org=36

we do:
http://localhost:3000/orgs/36/vaults

and even something like:

http://localhost:3000/orgs/36/circles/40/history

Reply:
i implemented this now, but not the nested circle under org, kinda outta scope for this